src/jsMain/kotlin/app/softwork/bootstrapcompose/Range.kt

```diff
@@ -0,0 +1,58 @@
+package app.softwork.bootstrapcompose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import kotlinx.uuid.UUID
+import org.jetbrains.compose.web.attributes.disabled
+import org.jetbrains.compose.web.attributes.forId
+import org.jetbrains.compose.web.dom.AttrBuilderContext
+import org.jetbrains.compose.web.dom.ContentBuilder
+import org.jetbrains.compose.web.dom.RangeInput
+import org.w3c.dom.HTMLInputElement
+import org.w3c.dom.HTMLLabelElement
+
+@Composable
+public fun Range(
+    value: Number,
+    min: Number? = null,
+    max: Number? = null,
+    step: Number = 1,
+    disabled: Boolean = false,
+    attrs: AttrBuilderContext<HTMLInputElement>? = null,
+    onInput: (Number?, HTMLInputElement) -> Unit,
+    content: @Composable RangeScope.() -> Unit = {}
```

Comment:
@Composable? If so, how does it affect the layout?

Reply:
Good question. I've run into a similar thing when working on the InputGroup. The easy path, which is what I am thinking is the right thing at this time, is to separate the labels from the component. This gives the user the freedom to layout the labels any way they want relative to the control. The downside is that the user will need to know to specify the same "forId" for the label as the "Id" for the control. A more difficult solution is to build into the API a way to cover all possible layout scenarios. But I think this will be complex, and likely better handled by a generalized layout API that is broadly applicable to all the components.